Rubrik/Public/Get-RubrikAPIToken.ps1

```diff
@@ -0,0 +1,79 @@
+#requires -Version 3
+function Get-RubrikAPIToken
+{
+  <#  
+      .SYNOPSIS
+      Connects to Rubrik and retrieves a list of generated API tokens
+
+      .DESCRIPTION
+      The Get-RubrikAPIToken cmdlet is used to pull a list of generated API tokens from the Rubrik cluster.
+
+      .NOTES
+      Written by Mike Preston for community usage
+      Twitter: @mwpreston
+      GitHub: mwpreston
+
+
+      .LINK
+     http://rubrikinc.github.io/rubrik-sdk-for-powershell/reference/Get-RubrikAPIToken.html
+
+      .EXAMPLE
```

Comment:
Each example should be separated with a new .EXAMPLE

Reply:
Fixed